eng/scripts/run_tests.ps1

```diff
@@ -32,8 +35,8 @@ if (Select-String -path ./report.xml -pattern '<testsuites></testsuites>' -simpl
     Get-Content ./coverage.json | gocov-xml > ./coverage.xml
     Get-Content ./coverage.json | gocov-html > ./coverage.html
 
-    Move-Item ./coverage.xml $repoRoot
-    Move-Item ./coverage.html $repoRoot
+    Move-Item -Force ./coverage.xml $repoRoot
```

Comment:
This will overwrite the file if it exists.  If the file exists can it be renamed first or prompted before overwriting?

Maybe its worth considering putting all output into directories that are specific to the run. i.e:
<rootdir>\builds\<timestamp>\

Reply:
I guess it's not clear to me how this script is used more generally. I know it's used in the pipeline, but do developers also use it for testing or prefer straight `go test` commands? I ran into this because I was running the script back to back in dev, and had to remove the file each time otherwise it would fail at the end.

We could back up the files, but if someone is just using it to test locally they may end up with a large number of files.

Reply:
Since it's just code coverage output archiving is probably not worth it - it's totally ephemeral and just rerunning tests will recreate it. I'm guessing this just moves it to the root for some other tool to pick it up later.

Reply:
@jhendrixMSFT , @gracewilcox , @chlowell  - do any of you use this script recreationally to run tests? I just use `go test` myself.

Reply:
I don't. Overwriting it should be fine.